code/game/objects/effects/overlays.dm

```diff
@@ -35,8 +35,7 @@
 	if(randomdir)
 		setDir(pick(cardinal))
 	flick("[icon_state]", src) //Because we might be pulling it from a pool, flick whatever icon it uses so it starts at the start of the icon's animation.
-	spawn(duration)
-		qdel(src)
+	QDEL_IN(src, duration)
```

Comment:
No, unless addtimer has a 1 decisecond check time and I'm pretty sure it's 2 deciseconds.
Otherwise stuff will be very slightly off and I'll be really annoyed about it